lib/meetups_data.rb

```diff
@@ -47,4 +48,79 @@ def self.from_hash(data)
       self.new(data)
     end
   end
+
+  class GroupOrganizer
+    attr_reader :data
+
+    def initialize(data)
+      @data = data
+    end
+
+    def find_organizers
+
+      return if !data["url"].include? 'http://www.meetup.com/'
+
+      return if has_organizers?
```

Comment:
It would probably be useful to add a flag that can be used to force an update of all organizers, otherwise changes will never be reflected.